mathics/builtin/forms.py

```diff
@@ -0,0 +1,520 @@
+"""
+Print Forms
```

Comment:
"Print Forms" seems wrong. I wouldn't call `InputForm`  a print form - although it can be used like that, its main purpose is for _input_.  `StringForm` is included here, and it too is not a print form, or not something listed in `$PrintForms`. 

Also, at the top it would be good to describe _something_ about what a form is.